### PR TITLE
[make_voxceleb1.pl] Fix train/test data split for the latest version of the voxceleb1 dataset 

### PR DESCRIPTION
**Pull request by sunshines14**
I found that previous scripts do not work with the latest version of the voxceleb1 dataset.
Therefore, I fixed the script for the latest version as follows: 
1. 'Url' update to download the latest version (from official site)
2. Delete the script about 'vox1_meta.csv' no longer needed
3. Fix the code for the latest version's output (variable name equals make_voxceleb2.pl)

It is confirmed that successful output can be obtained from the script. 
ex) voxceleb1_test -- spk2utt, trials, utt2spk, wav.scp

### CODE REVIEW

**Comment by sunshines14** [comment]
+ if previous script is still needed, I can use this modified script to another version (ex. v2) rather than replacing the existing script.

**Comment by danpovey** [comment]
Thanks a lot!! @david-ryan-snyder  can you please check this?

**Comment by david-ryan-snyder** [comment]
Thanks @sunshines14.

Could you do what you suggested and copy this to a new script called make_voxceleb1_v2.pl?

Then, in the run.sh script, make the new script the default one. Comment out the the old version of the script, and above it write a short comment describing the situation. E.g., mention that if you downloaded the dataset soon after it was released, you will want to use the make_voxceleb1.pl script instead.

File: egs/voxceleb/v1/local/make_voxceleb1.pl [hunk 1]
```diff
@@ -28,45 +28,28 @@
 closedir $dh;
 
 if (! -e "$data_base/voxceleb1_test.txt") {
-  system("wget -O $data_base/voxceleb1_test.txt http://www.openslr.org/resources/49/voxceleb1_test.txt");
-}
-
-if (! -e "$data_base/vox1_meta.csv") {
-  system("wget -O $data_base/vox1_meta.csv http://www.openslr.org/resources/49/vox1_meta.csv");
+  system("wget -O $data_base/voxceleb1_test.txt http://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test.txt");
```
Comment (david-ryan-snyder) [inline egs/voxceleb/v1/local/make_voxceleb1.pl]: Can you point the new URL to http://www.openslr.org/resources/49/voxceleb1_test_v2.txt instead? We prefer to host these things ourselves where possible.

**Comment by sunshines14** [comment]
Thanks @david-ryan-snyder.

I fixed some code as you commented as follow:
1. I have made a new script called 'make_voxceleb1_v2.pl'.
2. In the run.sh script, the v2 script used as the default one.
3. In addition, I have made the code simpler, including adding <dataset> in 'make_voxceleb1_v2.pl'.

It was reconfirmed that successful output can be obtained from all of fixed scripts.
Thanks.

File: egs/voxceleb/v1/local/make_voxceleb1_v2.pl [hunk 2]
```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/perl
+#
+# Copyright 2018  Ewald Enzinger
+#           2018  David Snyder
```
Comment (david-ryan-snyder) [inline egs/voxceleb/v1/local/make_voxceleb1_v2.pl]: You can credit yourself here, if you want.

File: egs/voxceleb/v1/run.sh [hunk 3]
```diff
@@ -24,11 +24,14 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script reates data/voxceleb1_test and data/voxceleb1_train.
+  # This script reates data/voxceleb1_test and data/voxceleb1_train for latest version of VoxCeleb1.
```
Comment (david-ryan-snyder) [inline egs/voxceleb/v1/run.sh]: Can you fix the typo, reates -> creates while you're at it?

**Comment by david-ryan-snyder** [comment]
Thanks @sunshines14! I just suggested you credit your work in the v2 perl script and fix a preexisting typo. Then we can merge it.

**Comment by sunshines14** [comment]
I did it all. 
Thanks @david-ryan-snyder.

File: egs/voxceleb/v1/local/make_voxceleb1_v2.pl [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+#           2019  Soonshin Seo
+#
+# Usage: make_voxceleb1.pl /export/voxceleb1 data/
```
Comment (david-ryan-snyder) [inline egs/voxceleb/v1/local/make_voxceleb1_v2.pl]: Sorry to nitpick, could you fix this comment, the usage is for the old script, not the new one.
Also, while you're here, could you write a comment that briefly summarizes why we need this script. Just say what you said before in the run.sh. Something like "The VoxCeleb 1 corpus underwent several updates that changed the directory and speaker ID format. The script make_voxceleb1.pl works for the oldest version of the corpus. This script should be used if you've downloaded the corpus recently."

Comment (sunshines14) [reply egs/voxceleb/v1/local/make_voxceleb1_v2.pl]: Fine. I agree with your opinion. I think it is an important point. 
Thanks.

**Comment by david-ryan-snyder** [comment]
Thanks @sunshines14, looks good to me. @danpovey, I think it's fine to merge this.

**Pull request merged**